src/components/Frame/Frame.scss

```diff
@@ -86,15 +90,32 @@ $skip-vertical-offset: rem(10px);
     opacity: 1;
   }
 
+  &:hover {
+    background-color: var(--p-overlay);
```

Comment:
This will apply when global theming is **not** enabled. Let's keep these changes so they only apply when global theming is enabled. We want to keep the old design language intact as we introduce the new one.